### PR TITLE
Add discourse to community drop-down

### PR DESCRIPTION
**Pull request by mribeirodantas**

### CODE REVIEW

File: templates/menu.ftl [hunk 1]
```diff
@@ -49,7 +49,8 @@
                         <li><a href="http://training.nextflow.io">Nextflow training</a></li>
                         <li><a href="http://nextflow-io.github.io/patterns/index.html">Implementation patterns</a></li>
                         <li><a href="https://github.com/nextflow-io/nextflow">GitHub repository</a></li>
-                        <li><a href="https://github.com/nextflow-io/nextflow/discussions">Discussion forum</a></li>
+                        <li><a href="https://community.seqera.io">Discourse forum</a></li>
```
Comment (ewels) [inline templates/menu.ftl:52]:
```suggestion
                        <li><a href="https://community.seqera.io">Community forum</a></li>
```

File: templates/menu.ftl [hunk 2]
```diff
@@ -49,7 +49,8 @@
...
+                        <li><a href="https://github.com/nextflow-io/nextflow/discussions">GitHub discussions</a></li>
                         <li><a href="https://www.nextflow.io/slack-invite.html">Slack community chat</a></li>
```
Comment (ewels) [inline templates/menu.ftl:54]: Up for discussion: should we remove these two links right away?

My preference is yes - remove.
```suggestion
```

Comment (mribeirodantas) [reply templates/menu.ftl:54]: Yeah, I would too, agree to remove them.

File: templates/menu.ftl [hunk 3]
```diff
@@ -49,7 +49,8 @@
                         <li><a href="http://training.nextflow.io">Nextflow training</a></li>
                         <li><a href="http://nextflow-io.github.io/patterns/index.html">Implementation patterns</a></li>
                         <li><a href="https://github.com/nextflow-io/nextflow">GitHub repository</a></li>
-                        <li><a href="https://github.com/nextflow-io/nextflow/discussions">Discussion forum</a></li>
+                        <li><a href="https://community.seqera.io">Community forum</a></li>
```
Comment (ewels) [inline templates/menu.ftl:52]: Should maybe link directly to the Nextflow category..

```suggestion
                        <li><a href="https://community.seqera.io/c/nextflow/5">Community forum</a></li>
```

File: templates/menu.ftl (hunk 2 above)
Comment (ewels) [reply templates/menu.ftl:54]: Not for now.

File: templates/menu.ftl (hunk 3 above)
Comment (mribeirodantas) [reply templates/menu.ftl:52]: 👍

**Review by ewels — APPROVED** [review]

**Pull request merged**